style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Times", sans-serif;
+}
+
+.wrapper {
+  width: 375px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  margin-left: 12px;
+  margin-top: 4px;
+}
+
+nav {
+  padding: 8px;
+}
+
+.menu-icon {
+  display: flex;
+  flex-direction: row;
+  width: 18px;
+  height: 18px;
+  margin-right: 100px;
+  color: #6070ff;
+  cursor: pointer;
+}
+
+section {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 114px 24px;
+  gap: 12px;
+}
+
+ul {
+  list-style: none;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/Header bg.png");
+  border-bottom-left-radius: 10px;
+}
+
+.headline-2 {
+  display: flex;
+  flex-direction: column;
+  background-image: none;
+  border-bottom-left-radius: 60px;
+}
+
+.title {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.details {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
```

Comment:
[OPTIONAL] please use the correct font-style and size for your website, I think it Poppins for your template, so that it is similar to the figma template, 
your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200120310-b1030656-6f8f-4510-be7b-83c18a3fb8d4.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200120341-4d82a6ad-339c-417b-8ac5-1f1a9b39f351.png)